agent/taskresource/credentialspec/credentialspec_windows_test.go

```diff
@@ -0,0 +1,110 @@
+// +build windows,unit
+
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package credentialspec
```

Comment:
I see more public accessible function in the tested file than those 3. Can you ensure they are properly covered by unit test?

Reply:
Add more tests to cover the exported and critical methods. Boilerplate code is not covered by unit tests.